### PR TITLE
adding info button that contains dialog box describing the function o…

### PR DESCRIPTION
**Pull request by melyxlin**
This PR adds info button to each of the module. Once the info button is clicked, an dialog box will show, describing the functionality of each module.

### CODE REVIEW

File: web/client/src/App.js [hunk 1]
```diff
@@ -22,25 +27,53 @@ function App() {
       <Grid item xs>
         <Grid container direction="column">
           <MaterialUI paperClass="paper-top">
-            <TokenForm parentCallback={callBackToken} />
+            <ModuleInfo
```
Comment (LeoRuiz01) [inline web/client/src/App.js:30]: I think we should also mention that a JWT Token can only be obtained with a service account key

Comment (LeoRuiz01) [reply web/client/src/App.js:30]: Also the command info does not work with a JWT token so that should be mentioned as well

Comment (melyxlin) [reply web/client/src/App.js:30]: Will put that into the token validation part

File: web/client/src/components/info/ModuleInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      <Dialog onClose={handleClose} open={open}>
+        <DialogTitle onClose={handleClose}>{title}</DialogTitle>
+        <DialogContent dividers>
+          {content.map((text, index) => (
+            <Typography key={index} gutterBottom>
+              {text}
+            </Typography>
+          ))}
+          {hasNote && (
+            <Typography variant="h6">
```
Comment (yousefa00) [inline web/client/src/components/info/ModuleInfo.jsx:54]: h6 was a bit loud, maybe we can keep this as the normal text size but keep the bolded "NOTE:" at the start?

**Comment by yousefa00** [comment]
@googlebot I consent.

**Pull request merged**